digital-twins/jsonld-contexts/digital_twins_compound.jsonld

```diff
@@ -1,6 +1,11 @@
 {
   "@context": [
     "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/usecase.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/airQualityMonitoring/jsonld-contexts/airQualityMonitoring.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/irrigation/jsonld-contexts/irrigation.jsonld",
```

Comment:
it already has pieces already present in this file. better to have one containing only irrigation terms.

Reply:
Not sure I understand ?

Reply:
at the end, you have:

```
         "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/digital-twins/jsonld-contexts/usecase.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/officeAirQuality/jsonld-contexts/airQuality-context.jsonld",
        "https://fiware.github.io/data-models/context.jsonld",
        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld"
        "https://raw.githubusercontent.com/smart-data-models/dataModel.Device/master/context.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/irrigation/jsonld-contexts/soil-attributes.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/irrigation/jsonld-contexts/observed-attributes.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/weather/jsonld-contexts/weather-compound.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/irrigation/jsonld-contexts/schema-attributes.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/authorization/jsonld-contexts/authorization.jsonld",
        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/alertAndAnomaly/jsonld-contexts/alertAndAnomaly.jsonld",
        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld"
    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/AlertAnomalyRisk/jsonld-contexts/AlertAnomalyRisk.jsonld",
    "https://raw.githubusercontent.com/smart-data-models/dataModel.RiskManagement/master/context.jsonld",
    "https://raw.githubusercontent.com/smart-data-models/dataModel.Alert/master/context.jsonld",
    "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld"
    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/mapping/jsonld-contexts/mapping.jsonld"
    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/authorization/jsonld-contexts/authorization.jsonld",
    "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.3.jsonld"
```

Reply:
it's guaranteed to have problems

Reply:
Should be better now